tests/python/tir-transform/test_tir_transform_convert_ssa.py

```diff
@@ -327,7 +327,8 @@ class TestDeDuplicateThreadIdxAcrossMultipleFunctions(BaseBeforeAfter):
     def before(self):
         threadIdx_x = tvm.tir.Var("threadIdx_x", "int32")
 
-        @I.ir_module
+        # complaints of duplicate definitions of threadIdx_x
+        @I.ir_module(check_well_formed=False)
```

Comment:
Is there anything that can be done with this case to make it well-formed?

Reply:
I think this one should disable the well formed check.  This test is validating that the pass has correct behavior when resolving SSA violations across multiple functions.  So, the `before` should be ill-formed, but the `expected` should be well-formed.

Reply:
Good to know, I will update the explanation